api/cpp/docs/cmake.md

```diff
@@ -1,56 +1,60 @@
 <!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
 <!-- cSpell: ignore ccmake dslint femtovg -->
 
-# Installing Or Building With CMake
+# Setup Developer Environment
 
-Slint comes with a CMake integration that automates the compilation step of the `.slint` markup language files and
-offers a CMake target for convenient linkage.
+## Prerequisite
 
-*Note*: We recommend using the Ninja generator of CMake for the most efficient build and `.slint` dependency tracking.
-Install [Ninja](https://ninja-build.org) and select the CMake Ninja backend by passing `-GNinja` or setting the `CMAKE_GENERATOR` environment variable to `Ninja`.
+* A C++ compiler that supports C++20 (e.g., **MSVC 2019 16.6** on Windows)
+
+* **[cmake](https://cmake.org/download/)** (3.21 or newer)
+
+  * Slint comes with a CMake integration that automates the compilation step of the `.slint` markup language files and offers a CMake target for convenient linkage.
+
+  * *Note*: We recommend using the Ninja generator of CMake for the most efficient build and `.slint` dependency tracking. Install [Ninja](https://ninja-build.org) and select the CMake Ninja backend by passing `-GNinja` or setting the `CMAKE_GENERATOR` environment variable to `Ninja`.
+
+## Setup Slint
+
+To setup Slint, you can either download the [binary packages](#install-binary-packages) or [build from sources](#build-from-sources).  
```

Comment:
(also removed the "can", it's not just an option, one or the other must be selected)

```suggestion
To install Slint, either download the [binary packages](#install-binary-packages) or [build from sources](#build-from-sources).  
```